siwiflasher.py

```diff
@@ -373,19 +373,19 @@ def da_mem(self, address, size, ftype, file_count=1, fota=NACK):
             ASSERT(r == ACK, "DA_MEM ACK")
 
         r = struct.unpack(">BB", self.send(NONE, 2)) #filecount + ACK
-        ASSERT(r[0] == file_count, "File count does not match")
-        format_acks = 0
+        #ASSERT(r[0] == file_count, "File count does not match")
```

Comment:
Shouldn't we keep this for sanity check?

Reply:
For some reason my standard VIVA image is returning 2 instead of 1
So when sending the ROM + VIVA + APP I receive 4 instead of 3
Everything else is working fine